[2022-02-09 08:58:50,357] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 08:58:50,374] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 08:58:50,374] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:58:50,374] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:58:50,375] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:58:50,395] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-05-01 23:00:00+00:00
[2022-02-09 08:58:50,403] {standard_task_runner.py:52} INFO - Started process 131 to run task
[2022-02-09 08:58:50,412] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-05-01T23:00:00+00:00', '--job-id', '451', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpycep500e', '--error-file', '/tmp/tmphy22dfd3']
[2022-02-09 08:58:50,413] {standard_task_runner.py:77} INFO - Job 451: Subtask local_to_gcs_task
[2022-02-09 08:58:50,518] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-05-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:58:50,627] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:58:50,666] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-01T23:00:00+00:00
[2022-02-09 08:58:54,873] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 08:58:54,887] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20200501T230000, start_date=20220209T085850, end_date=20220209T085854
[2022-02-09 08:58:54,942] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:58:54,990] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:13:59,747] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:13:59,765] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:13:59,766] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:13:59,766] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:13:59,766] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:13:59,787] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-05-01 23:00:00+00:00
[2022-02-09 09:13:59,796] {standard_task_runner.py:52} INFO - Started process 190 to run task
[2022-02-09 09:13:59,810] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-05-01T23:00:00+00:00', '--job-id', '539', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpsxrkyzhy', '--error-file', '/tmp/tmpqq43yk1z']
[2022-02-09 09:13:59,811] {standard_task_runner.py:77} INFO - Job 539: Subtask local_to_gcs_task
[2022-02-09 09:13:59,908] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-05-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:13:59,972] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:00,010] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-01T23:00:00+00:00
[2022-02-09 09:14:03,646] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:14:03,709] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20200501T230000, start_date=20220209T091359, end_date=20220209T091403
[2022-02-09 09:14:03,771] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:03,828] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
